src/ansys/templates/python/doc_project/{{cookiecutter.__project_name_slug}}/README.rst

```diff
@@ -0,0 +1,131 @@
+{{ cookiecutter.__project_name_slug }}
+{{ '=' * (cookiecutter.__project_name_slug | length) }}
+
+{{ cookiecutter.__short_description }}
+
+
+How to install
+--------------
+
+At least two installation modes are provided: user and developer.
+
+For users
+^^^^^^^^^
+
+User installation can be performed by running:
+
+.. code:: bash
+
+    python -m pip install {{ cookiecutter.__pkg_name }}
+
+For developers
+^^^^^^^^^^^^^^
+
+Before contributing to the project, please refer to the `PyAnsys Developer's
+guide`_. You will need to follow these steps:
+
+1. Start by cloning this repository:
+
+    .. code:: bash
+
+        git clone {{ cookiecutter.repository_url }}
+
+2. Create a fresh-clean Python environment and activate it. Refer to the
+   official `venv`_ documentation if you require further information:
+
+    .. code:: bash
+
+        # Create a virtual environment
+        python -m venv .venv
+
+        # Activate it in a POSIX system
+        source .venv/bin/activate
+
+        # Activate it in Windows CMD environment
+        .venv\Scripts\activate.bat
+
+        # Activate it in Windows Powershell
+        .venv\Scripts\Activate.ps1
+
+3. Make sure you have the latest version of `pip`_:
+
+    .. code:: bash
+
+        python -m pip install -U pip
+
+4. Install the project in editable mode:
+
+    .. code:: bash
+    
+        python -m pip install --editable {{ cookiecutter.__pkg_name }}
+
+5. Install additional requirements (if needed):
+
+     .. code:: bash
+
+        python -m pip install -r requirements_build.txt
+        python -m pip install -r requirements_doc.txt
+        python -m pip install -r requirements_tests.txt
+
+
+6. Finally, verify your development installation by running:
+
+    .. code:: bash
+        
+        python -m pip install -r requirements_tests.txt
+        pytest tests -vv
+
```

Comment:
Use automatically generated lists with rst
```suggestion

#. Start by cloning this repository:

    .. code:: bash

        git clone {{ cookiecutter.repository_url }}

#. Create a fresh-clean Python environment and activate it. Refer to the
   official `venv`_ documentation if you require further information:

    .. code:: bash

        # Create a virtual environment
        python -m venv .venv

        # Activate it in a POSIX system
        source .venv/bin/activate

        # Activate it in Windows CMD environment
        .venv\Scripts\activate.bat

        # Activate it in Windows Powershell
        .venv\Scripts\Activate.ps1

#. Make sure you have the latest version of `pip`_:

    .. code:: bash

        python -m pip install -U pip

#. Install the project in editable mode:

    .. code:: bash
    
        python -m pip install --editable {{ cookiecutter.__pkg_name }}

#. Install additional requirements (if needed):

     .. code:: bash

        python -m pip install -r requirements_build.txt
        python -m pip install -r requirements_doc.txt
        python -m pip install -r requirements_tests.txt


#. Finally, verify your development installation by running:

    .. code:: bash
        
        python -m pip install -r requirements_tests.txt
        pytest tests -vv

```

Reply:
This improves the maintanability of such enumerated steps... We should try and implement them everywhere

Reply:
Ah, I didn't know about this syntax. Thanks for pointing this, @RobPasMue, I truly see its advantages. No longer need to worry about manually writing ordinals.

Reply:
My pleasure! We should start implementing it wherever we see it. Manually writing list numbers is not maintainable.